completion/pdfMsym.cwl

```diff
@@ -0,0 +1,161 @@
+# pdfMsym package
```

Comment:
the file is `pdfmsym.tex` so even though this cwl would have to be manually turned on in the settings (only .sty and .cls files are picked up automatically), I think it would be better to name it `pdfmsym.cwl`